engine/src/test/java/io/zeebe/engine/processing/workflowinstance/CreateWorkflowInstanceProcessorTest.java

```diff
@@ -85,7 +82,7 @@ public void setUp() {
 
     processor =
         new CreateWorkflowInstanceProcessor(
-            workflowState, elementInstanceState, variablesState, keyGenerator);
+            workflowState, elementInstanceState, variablesState, keyGenerator, writers);
```

Comment:
In this class there is an unchanged line. The annotation at the top. I think we can replace `@RunWith(MockitoJUnitRunner.StrictStubs.class)` with `@RunWith(MockitoJUnitRunner.class)` as strict stubbing is enabled by default now. Same for `UpdateVariableDocumentProcessorTest`